src/main/java/org/jabref/logic/formatter/bibtexfields/NormalizeEmDashesIEEEFormatter.java

```diff
@@ -0,0 +1,31 @@
+package org.jabref.logic.formatter.bibtexfields;
+
+import org.jabref.logic.cleanup.Formatter;
+import org.jabref.logic.l10n.Localization;
+
+public class NormalizeEmDashesIEEEFormatter extends Formatter {
+    @Override
+    public String getName() {
+        return Localization.lang("Normalize em dashes from IEEE");
+    }
+
+    @Override
+    public String getKey() {
+        return "normalize_em_dash_ieee";
+    }
+
+    @Override
+    public String format(String value) {
+        return value.replaceAll("&amp;#x2014;", "—");
```

Comment:
Seeing the input, isnt that a double encoded unicode character? 

Should the importer run https://github.com/JabRef/jabref/blob/main/src/main/java/org/jabref/logic/formatter/bibtexfields/HtmlToUnicodeFormatter.java twice for the title field?

Reply:
But that will leave the ampersand.

I've tried to add additional [HtmlToUnicodeFormatter.java](https://github.com/JabRef/jabref/blob/main/src/main/java/org/jabref/logic/formatter/bibtexfields/HtmlToUnicodeFormatter.java) to `DoiFetcher.doPostCleanup`. Is that the right place? 

It replaced the `&amp;` with ampersand, but `#x2014;` was left as is. So, probably there should be a separate formatter?

I read the original issue and you have provided a sketch of the solution. You said to add a new cleanup class and integrate it with `IEEE` class. But how to add the cleanup to the `IEEE`? There seems to be no `doPostCleanup` method. What if we just change the `parseJsonResponse` and directly cleanup from there?

Reply:
For the IEEE doPostCleanup  method, it's  just a  single method that is called manually because only the EntryBasedParserFetcher has it defined as interface method.
See thee DoiFetcher for example, it defines it manually

Reply:
It seems that the JabRef doesn't use `IEEE` class when importing the DOI [https://doi.org/10.1109/PERCOMW.2015.7133989](https://doi.org/10.1109/PERCOMW.2015.7133989)